packages-native/popup-menu/src/PopupMenu.tsx

```diff
@@ -29,50 +28,52 @@ export function PopupMenu(props: PopupMenuProps<PopupMenuStyle>): ReactElement {
         [menuRef.current]
     );
 
-    const Touchable: ComponentType<TouchableNativeFeedbackProps | TouchableHighlightProps> =
-        Platform.OS === "android" ? TouchableNativeFeedback : TouchableHighlight;
+    const Touchable: ComponentType<TouchableNativeFeedbackProps | TouchableOpacity> =
+        Platform.OS === "android" ? TouchableNativeFeedback : TouchableOpacity;
 
     let menuOptions: ReactElement[];
     if (props.renderMode === "basic") {
-        menuOptions = props.basicItems.map((item, index) =>
-            item.itemType === "divider" ? (
-                <MenuDivider key={index} color={styles.basicItem?.dividerColor} />
+        menuOptions = props.basicItems.map((item, index) => {
+            const itemStyle = styles.basic?.itemStyle && styles.basic?.itemStyle[item.styleClass];
+            return item.itemType === "divider" ? (
+                <MenuDivider key={index} color={styles.basic?.dividerColor} />
             ) : (
                 <MenuItem
                     key={index}
                     onPress={() => handlePress(item.action)}
-                    underlayColor={styles.basicItem?.underlayColor}
-                    textStyle={styles.basicItem?.textStyle}
-                    ellipsizeMode={styles.basicItem?.ellipsizeMode as any}
-                    style={styles.basicItem?.container as any}
+                    textStyle={itemStyle}
+                    ellipsizeMode={styles.basic?.itemStyle?.ellipsizeMode as any}
```

Comment:
Cant you for the type here?

Reply:
My MR is still not merged so this one still expects "head" | "tail", although its not true

Reply:
Fine